scene/gui/tabs.cpp

```diff
@@ -120,7 +120,6 @@ void Tabs::gui_input(const Ref<InputEvent> &p_event) {
 		}
 
 		_update_hover();
-		update();
```

Comment:
`_update_hover` doesn't call `update()` would guess that this is still needed.

Reply:
When I tried it, I couldn't see any difference between the update() being there and not being there, but I don't know enough about the software to be sure I checked every aspect of the tabbing. I shall add it back in just to be safe.